Source/Blazorise/Components/Droppable/DropZone.razor.cs

```diff
@@ -317,20 +332,38 @@ private async Task OnDropHandler()
             return (item, canBeDropped);
         }
 
-        private void OnDragStarted() => dragging = true;
+        private Task OnDragStarted()
+        {
+            dragging = true;
 
-        private void OnDragEnded() => dragging = false;
+            return Task.CompletedTask;
+        }
 
-        private IEnumerable<TItem> GetItems()
+        private Task OnDragEnded( TItem item )
         {
-            Func<TItem, bool> predicate = ( item ) => ParentContainer.ItemsFilter( item, Name ?? string.Empty );
+            dragging = false;
 
-            if ( ItemsFilter != null )
+            return Task.CompletedTask;
+        }
+
+        private int count;
+        private IEnumerable<TItem> GetItems()
+        {
+            if ( recalculateItems )
```

Comment:
We've used something like `Dirty`, `IsDirty` coventions on other places I believe? `Datagrid` comes to mind, do we want to maintain same convention? 
Otherwise `recalculateItems` seems good.